materials/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+
+
+class Course(models.Model):
+    """Класс курс"""
+
+    name = models.CharField(
+        max_length=255, verbose_name="Название курса", help_text="Укажите курс"
+    )
+    preview = models.ImageField(
+        upload_to="materials/preview", verbose_name="Превью", blank=True, null=True
+    )
+    description = models.TextField(
+        blank=True,
+        null=True,
+        verbose_name="Описание курса",
+        help_text="Укажите описание курса",
+    )
+
+    class Meta:
+        verbose_name = "Курс"
+        verbose_name_plural = "Курсы"
+
+
+class Lesson(models.Model):
+    """Класс урок"""
+
+    name = models.CharField(
+        max_length=255, verbose_name="Название урока", help_text="Укажите урок"
+    )
+    courses = models.ForeignKey(
```

Comment:
Название поля для FK должно быть в единственном числе
```suggestion
    course = models.ForeignKey(
```